xbmc/pvr/channels/PVRChannelsPath.cpp

```diff
@@ -69,7 +69,15 @@ CPVRChannelsPath::CPVRChannelsPath(const std::string& strPath)
         std::vector<std::string> tokens = StringUtils::Split(segment, "_");
         if (tokens.size() == 2)
         {
-          m_clientID = tokens[0];
+          std::vector<std::string> instance = StringUtils::Split(tokens[0], "@");
```

Comment:
Please fix this comment "// pvr://channels/(tv|radio)/<groupname>/<addonid>_<channeluid>.pvr" some lines below to reflect the extended syntax.